target/linux/realtek/files-5.15/drivers/net/dsa/rtl83xx/rtl838x.c

```diff
@@ -1730,7 +1730,7 @@ const struct rtl838x_reg rtl838x_reg = {
 	.vlan_fwd_on_inner = rtl838x_vlan_fwd_on_inner,
 	.set_vlan_igr_filter = rtl838x_set_igr_filter,
 	.set_vlan_egr_filter = rtl838x_set_egr_filter,
-	.enable_learning = rtl838x_enable_learning,
+//	.enable_learning = rtl838x_enable_learning,
```

Comment:
```suggestion
//	.enable_learning = rtl838x_enable_learning,  FIXME: L2 learning is broken on 5.15, disable for now
```
Lets add the comment so everyone understands why this is turned off. If you think the comment doesn't serve any purpose, then don't commit commented code to begin with, and just delete it all together :p your call (Personally, I'd keep the commented function, as we may not merge this with this regression?)